### PR TITLE
 Add to user map - Mar 2016 #5158

### LINKED ISSUE #5158: Add to user map - Mar 2016
Add the following institutes (who joined TEAMMATES in March 2016) to the user map
```
La Trobe University	Australia
University of Queensland	Australia
All Saints' College	Australia
University of Canberra	Australia
Universidade Catolica de Brasilia	Brazil
Queen's University	Canada
University of British Columbia	Canada
Simon Fraser University	Canada
Clark University	China
Erhvervsakademiet Lillebælt	Denmark
The International School of Elite education	Egypt
Modern Education Schools	Egypt
The International School of Elite education	Egypt
Al-Hoda International School	Egypt
Al-Alyaa Private School	Egypt
IESEG	France
EMLYON Business School	France
Gottfried Wilhelm Leibniz Universität Hannover	Germany
CCC Rotary Secondary School	Hong Kong
Codecool	Hungary
Bharati Vidypeeth College of Engineering, New Delhi	India
USICT, New Delhi	India
IIT Bombay	India
eLearning Centre Seeu	Macedonia
Univerisiti Malaysia Sarawak	Malaysia
Universiti Tunku Abdul Rahman	Malaysia
Utrecht University	Netherlands
Rotterdam School of Management	Netherlands
Hogeschool van Amsterdam	Netherlands
Scala College Alphen aan den Rijn	Netherlands
University of Auckland	New Zealand
Universidad San Carlos	Paraguay
Saint Louis University	Philippines
Higher School of Economics	Russia
Higher School of Economic	Russia
Singapore Institute of Technology	Singapore
Upf	Spain
University of Moratuwa	Sri Lanka
Haute Ecole de Gestion de Genève	Switzerland
Haute Ecole de Gestion de Genève	Switzerland
Balikesir University	Turkey
University of Warwick	UK
London School of Economics and Political Science	UK
London School of Economics and Political Science	UK
Hult International Business School	UK
University of Surrey	UK
Kingston University London	UK
San Jose State University	USA
University of Texas at Austin	USA
California State University Long Beach	USA
University of Texas at Arlington	USA
Westmont College	USA
Stanford University	USA
Front Range Community College	USA
Garinger High School	USA
Aurora University	USA
Drexel University	USA
Merrimack College	USA
Caledonia Middle School	USA
Unniversity of Washington, Seattle	USA
Hackensack High School	USA
Ohio University 	USA
Radford University	USA
Hult International Business School	USA
Columbia Basin College	USA
Xavier University	USA
Rutgers Robert Wood Johnson Medical School	USA
Hult International Business School	USA
Walsh University	USA
University of Texas Arlington	USA
New Mexico State University	USA
Caledonia Middle School	USA
Sequoyah High School	USA
Rensselaer Polytechnic Institute	USA
Davis Senior High School	USA
```

### PR DESCRIPTION
**Pull request by chowyb**
Fixes #5158 

The following non-exact duplicates were not added to the user map:
```
Higher School of Economic   Russia
Universidade Catolica de Brasilia   Brazil
IESEG   France
```
Any duplicates with an exact match were not added to the user map.

### CODE REVIEW

File: src/main/webapp/js/userMapData.json [hunk 1]
```diff
@@ -304,8 +328,11 @@
 ["Universidad Pública de Navarra","Spain"],
 ["Universitat Autonoma de Barcelona","Spain"],
 ["University of the Basque Country","Spain"],
+["Upf","Spain"],
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: This one looks dubious. let's remove it.

File: src/main/webapp/js/userMapData.json [hunk 2]
```diff
@@ -250,6 +273,7 @@
 ["Carnegie Mellon University","Qatar"],
 ["Qatar University","Qatar"],
 ["Deltacredit","Russia"],
+["Higher School of Economics","Russia"],
 ["National Research University","Russia"],
 ["NRU-Higher School of Economics","Russia"],
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: This looks same as the one we added above. May be remove this one and keep the one we added?

File: src/main/webapp/js/userMapData.json [hunk 3]
```diff
@@ -304,8 +328,11 @@
 ["Universidad Pública de Navarra","Spain"],
 ["Universitat Autonoma de Barcelona","Spain"],
 ["University of the Basque Country","Spain"],
+["Upf","Spain"],
+["University of Moratuwa","Sri Lanka"],
 ["Business School Lausanne","Switzerland"],
 ["EPFL","Switzerland"],
+["Haute Ecole de Gestion de Genève","Switzerland"],
 ["HEG Genève","Switzerland"],
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: These two look the same. Keep the new and and remove the old one?

File: src/main/webapp/js/userMapData.json [hunk 4]
```diff
@@ -332,9 +360,12 @@
 ["Edinburgh Napier University","UK"],
 ["Emmanuel College","UK"],
 ["Home Learning College","UK"],
+["Hult International Business School","UK"],
 ["Imperial College London","UK"],
+["Kingston University London","UK"],
 ["Leeds Trinity University","UK"],
 ["London School of Economics","UK"],
+["London School of Economics and Political Science","UK"],
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Check online to see if these are two different institutes.

File: src/main/webapp/js/userMapData.json [hunk 5]
```diff
@@ -482,6 +521,7 @@
 ["Hot Springs Intermediate School","USA"],
 ["Howard Payne University","USA"],
 ["Hudson County Community College","USA"],
+["Hult International Business School","USA"],
 ["Hult International School of Business","USA"],
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: These two are the same. Keep whichever the one used by their website.

File: src/main/webapp/js/userMapData.json [hunk 6]
```diff
@@ -616,6 +658,8 @@
 ["Santa Clara University","USA"],
 ["Santa Fe College","USA"],
 ["Scott Community College","USA"],
+["SCU","USA"],
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Remove this. It was a spam account.

File: src/main/webapp/js/userMapData.json [hunk 7]
```diff
@@ -722,6 +767,7 @@
 ["University of Tennessee","USA"],
 ["University of Tennessee Knoxville","USA"],
 ["University of Texas","USA"],
+["University of Texas Arlington","USA"],
 ["University of Texas at Arlington","USA"],
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: These two are the same.

File: src/main/webapp/js/userMapData.json [hunk 8]
```diff
@@ -772,6 +821,7 @@
 ["Worcester Polytechnic Institute","USA"],
 ["Wright State University","USA"],
 ["WSU Vancouver","USA"],
+["Xavier University","USA"],
 ["Xavier University of Louisiana","USA"],
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Check if these two are different.

**Comment by damithc** [comment]
Some changes requested. Let us know when ready for review again.

File: src/main/webapp/js/userMapData.json (hunk 1 above)
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Removed

File: src/main/webapp/js/userMapData.json (hunk 2 above)
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Removed `NRU-Higher School of Economics`

File: src/main/webapp/js/userMapData.json (hunk 3 above)
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Removed `HEG Genève`

File: src/main/webapp/js/userMapData.json (hunk 4 above)
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: These refer to the same institute, although `London School of Economics` seems to be a common way to refer to the institute. Kept `London School of Economics and Political Science` as based off their website.

File: src/main/webapp/js/userMapData.json (hunk 5 above)
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Kept `Hult International Business School`

File: src/main/webapp/js/userMapData.json (hunk 6 above)
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Removed

File: src/main/webapp/js/userMapData.json (hunk 7 above)
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Kept `University of Texas at Arlington`

File: src/main/webapp/js/userMapData.json (hunk 8 above)
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: These two institutes are different.

`Xavier University` is based in Cincinnati, Ohio;
`Xavier University of Louisiana` is based in New Orleans, Louisiana.

Both entries kept.

**Comment by chowyb** [comment]
@damithc ready for review

File: src/main/webapp/js/userMapData.json [hunk 9]
```diff
@@ -132,16 +142,20 @@
...
 ["MIO-ECSDE","Greece"],
 ["Colegio Americano del Sur","Guatemala"],
+["CCC Rotary Secondary School","Hong Kong"],
 ["HKU SPACE Community College","Hong Kong"],
 ["Queen Elizabeth School","Hong Kong"],
 ["The Hong Kong Polytechnic University","Hong Kong"],
 ["University of Hong Kong","Hong Kong"],
+["Codecool","Hungary"],
 ["All India Institute of Medical Sciences","India"],
+["Bharati Vidypeeth College of Engineering, New Delhi","India"],
```
Comment (thyageshm) [inline src/main/webapp/js/userMapData.json]: I think this one should be Vidy**a**peeth and not Vidypeeth. Please re-check all the spellings once

Comment (thyageshm) [inline src/main/webapp/js/userMapData.json]: actually @chowyb wait. 

@damithc should we edit this? I think you have generated the list based on the institute names given by the instructors themselves right? Maybe we should leave it as it is....

**Comment by thyageshm** [comment]
Just a minor clarification, then we can merge this

File: src/main/webapp/js/userMapData.json (hunk 9 above)
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: It's ok, go ahead and correct the typo.

File: src/main/webapp/js/userMapData.json (hunk 8 above)
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Got it.

**Comment by chowyb** [comment]
Edits made as requested. I have looked up each of the institutes on the Internet to find their correct spelling.

As an additional note, I would like to bring up two institutes, the first of which is: `Hult International Business School`. With the new data, it is now appearing 3 times in our data in the following locations: `China, UK, USA`. Their website structure suggests that this is one institute with multiple campuses in different countries.

The second institute is `Clark University`, which based off the data is stated to be in China. However, this university is in USA, but offers special programs to study in China.

I am unsure if any of these will require further action.

**Comment by damithc** [comment]
Thanks for checking up those cases @chowyb. Let's keep both as they are, because different campuses of universities operate fairly independently despite having the same name.

**Comment by chowyb** [comment]
Made some final changes to the list (sorting the updated names, removing an additional duplicate) and merging the latest version of master for successful build on Travis, ready for review.

**Comment by thyageshm** [comment]
Great work @chowyb!

**Comment by damithc** [comment]
@thyageshm can try the online merge for this one

**Comment by thyageshm** [comment]
ok!

**Comment by thyageshm** [comment]
The merge button is still the old one... not sure what the problem is.

**Comment by damithc** [comment]
You have to click the button first.

**Comment by thyageshm** [comment]
@damithc I have merged this one using the online tool and it worked well. I made a mistake and included the pr commit messages as part of the commit body (the subject line follows our `[issue#] issuetitle` format.

**Pull request merged**